codec/seata-codec-hessian/src/main/java/io/seata/codec/hessian/HessianCodec.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.codec.hessian;
+
+import com.caucho.hessian.io.Hessian2Input;
+import com.caucho.hessian.io.Hessian2Output;
+import com.caucho.hessian.io.Serializer;
+import com.caucho.hessian.io.SerializerFactory;
+import io.seata.common.loader.LoadLevel;
+import io.seata.core.codec.Codec;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+/**
+ * @Xin Wang
+ */
+@LoadLevel(name = "HESSIAN")
+public class HessianCodec implements Codec {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HessianCodec.class);
+
+    @Override
+    public <T> byte[] encode(T t) {
+        byte[] stream = null;
+        SerializerFactory hessian = HessianSerializerFactory.getInstance();
+        try {
+            Serializer serializer = hessian.getSerializer(t.getClass());
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            Hessian2Output output = new Hessian2Output(baos);
+            serializer.writeObject(t, output);
+            output.close();
+            stream = baos.toByteArray();
+        } catch (IOException e) {
+            LOGGER.error("Hessian encode error", e);
+        }
+        return stream;
+    }
+
+    @Override
+    public <T> T decode(byte[] bytes) {
+        T obj = null;
+        try (ByteArrayInputStream is = new ByteArrayInputStream(bytes);) {
+            Hessian2Input input = new Hessian2Input(is);
```

Comment:
It's better to keep Hession2Input in the try() ,because ByteArrayInputStream does nothing with it's close method.
![image](https://user-images.githubusercontent.com/24457941/70298221-d38b3f00-182b-11ea-9ab2-cc44786eeb0c.png)